tests/foreman/ui/test_bookmarks.py

```diff
@@ -409,16 +565,26 @@ def test_positive_delete_bookmark(self):
         3. Verify the bookmark is no longer listed
 
         @Assert: The bookmark is deleted
-
-        @Status: Manual
         """
-        for entity in BOOKMARK_ENTITIES:
-            if entity.get('setup'):
-                create_entity(entity['setup'])
-            pass
+        for entity in self.entities:
+            with self.subTest(entity):
+                with Session(self.browser) as session:
+                    name = gen_random_string()
+                    session.nav.go_to_select_org(self.org_.name)
+                    ui_lib = getattr(self, entity['name'].lower())
+                    ui_lib.create_a_bookmark(
+                        name=name,
+                        public=True,
+                        query=gen_random_string(),
+                    )
+                    self.assertIsNotNone(
+                        self.bookmark.search(entity['controller'], name))
+                    self.bookmark.delete(entity['controller'], name)
+                    self.assertIsNone(
```

Comment:
Is not required in case you are already put additional verification into your class `delete` method. It was necessary to work in that way to decrease overall time for run. If you create your custom method, I suggest to leave that assert, but simplify your `delete` method as these few seconds that we save is not worth to have so complex initial method